Makefile

```diff
@@ -0,0 +1,14 @@
+# Building docker image
+VERSION = "v1.0.0"
+IM = shalsh/phenoscape-pipeline-tools
+
+build:
+	docker build -t $(IM):$(VERSION) . \
+	&& docker tag $(IM):$(VERSION) $(IM):latest
```

Comment:
> I don't think you need the `&&` concatenation of the commands here? Make should do this (stopping in case of error) on its own already.

This may be my influence, which I picked up from @cmungall's example. I think you are right though, I just did a test.